lib/osf-components/addon/components/editable-field/publication-doi-manager/component.ts

```diff
@@ -0,0 +1,106 @@
+import { tagName } from '@ember-decorators/component';
+import { action, computed } from '@ember-decorators/object';
+import { alias, and, not } from '@ember-decorators/object/computed';
+import { service } from '@ember-decorators/service';
+import Component from '@ember/component';
+import EmberObject from '@ember/object';
+import { task } from 'ember-concurrency';
+import { buildValidations, Validations, validator } from 'ember-cp-validations';
+import I18N from 'ember-i18n/services/i18n';
+import Toast from 'ember-toastr/services/toast';
+
+import { layout } from 'ember-osf-web/decorators/component';
+import Registration from 'ember-osf-web/models/registration';
+import extractDoi, { DOIRegex } from 'ember-osf-web/utils/extract-doi';
+import template from './template';
+
+export interface PublicationDoiManager {
+    save: () => void;
+    publicationDoi: string;
+    inEditMode: boolean;
+    validationNode: Registration & Validations;
+    didValidate: boolean;
+}
+
+const DoiValidations = buildValidations({
+    articleDoi: [
+        validator('format', {
+            regex: DOIRegex,
```

Comment:
Seems like this shouldn't be required. Users should be able to set it blank if they want.

Reply:
Set it blank so as to delete an existing pub DOI?

Reply:
Yeah. On preprints, users have sometimes misunderstood the point of the field and given the DOI we minted for this exact registration, or for something else they think is related but isn't a published version of this registration. They should be able to go back to no publication DOI if they need to.